docs/dependency-specification.md

```diff
@@ -172,6 +172,7 @@ how [repository credentials]({{< relref "repositories#configuring-credentials" >
 ```bash
 poetry config repositories.git-org-project https://github.com/org/project.git
 poetry config http-basic.git-org-project username token
+poetry add git+http://private-repo.example.com/project.git
```

Comment:
This is not the right place to demonstrate this at all -- it would be better off somewhere else (e.g. `basic-usage` or `commands`), or just not mentioned at all.